x/mongo/driver/operation.go

```diff
@@ -48,6 +48,12 @@ var (
 	ErrNonPrimaryReadPref = errors.New("read preference in a transaction must be primary")
 	// errDatabaseNameEmpty occurs when a database name is not provided.
 	errDatabaseNameEmpty = errors.New("database name cannot be empty")
+	// errEmptyWriteConcern indicates that a write concern has no fields set.
+	errEmptyWriteConcern = errors.New("a write concern must have at least one field set")
+	// errNegativeW indicates that a negative integer `w` field was specified.
+	errNegativeW = errors.New("write concern `w` field cannot be a negative number")
+	// errInconsistent indicates that an inconsistent write concern was specified.
+	errInconsistent = errors.New("a write concern cannot have both w=0 and j=true")
```

Comment:
These errors were taken from the write concern package, and will be removed from writeconcern in GODRIVER-2821